server/src/main/java/org/elasticsearch/common/hash/Murmur3Hasher.java

```diff
@@ -20,6 +20,8 @@
  */
 public class Murmur3Hasher {
 
+    public static final String METHOD = "MurmurHash3_x64_128";
```

Comment:
Do we expect to add a few different murmur hash types? This is a pretty awkward name, but it makes sense if we expect to add other configurations

Reply:
I'm not aware of any plans to add additional variants of the Murmur hash, so I think we could shorten it to `MurmurHash3` as it's commonly known, possibly with a note in the docs that it's the x64/128 variant that's used.